### PR TITLE
Cleaning up storybook files

### PR DESCRIPTION
**Pull request by sean-brydon**
Hello! 

With help of @JeroenReumkens we have been working on creating proper design documentation to match our Figma as close as possible.  During this time we are also starting to migrate away from V2 folder into a `@calcom/ui/components` This will be the final directory of all UI components. Once they have undergone review - and also had the relevant documentation created for them. 

As you can see we have only migrated 5 components in this PR and there are a lot of file changes.  So we're gonna push for this bulk to get reviewed and merged. From here we will work on doing it as a simple component at the time to keep the amount of changes to the code base to a minimum. 

We dropped a readme in the ui folder to explain what's going on in some more detail. 
CleanShot 2022-11-02 at 13 26 47

https://www.loom.com/share/df708fdac46b470ea201ef632b1f30a6

TODO: 

- [X] Update website, console, api with new paths
- [X] Delete v1 versions of new components

These need to be promoted to production when this PR goes live. 
Website: https://github.com/calcom/website/pull/393
Console: https://github.com/calcom/console/pull/39

### CODE REVIEW

**Review by PeerRich — APPROVED** [review]

File: apps/storybook/.gitignore [hunk 1]
```diff
@@ -1,38 +1,24 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (JeroenReumkens) [inline apps/storybook/.gitignore:1]: As for storybook, we deleted any old storybook config and stories. So if you see deletes of stories, that's why. And diffs in config main that we created a new file, which still had the same name.

File: apps/storybook/tailwind.config.js [hunk 2]
```diff
@@ -1,167 +1,7 @@
...
-      maxWidth: (theme, { breakpoints }) => ({
-        0: "0",
-        ...theme("spacing"),
-        ...breakpoints(theme("screens")),
-        full: "100%",
-        screen: "100vw",
-      }),
-    },
-  },
+  ...base,
```
Comment (JeroenReumkens) [inline apps/storybook/tailwind.config.js:4]: Storybook now uses the same config as all the other places ✌️

**Review by emrysal — APPROVED** [review]
Had a once over console and website too and tested through various points of this PR, approving it, but it's very big. Smaller more contained PRs 👍

**Comment by sean-brydon** [comment]
> Had a once over console and website too and tested through various points of this PR, approving it, but it's very big. Smaller more contained PRs 👍

One component at a time from now :) I promise

**Pull request merged**